esmvalcore/preprocessor/_supplementary_vars.py

```diff
@@ -65,47 +78,62 @@ def add_cell_measure(cube, cell_measure_cube, measure):
         raise ValueError(
             f"measure name must be 'area' or 'volume', got {measure} instead"
         )
-    measure = iris.coords.CellMeasure(
-        cell_measure_cube.core_data(),
+    coord_dims = tuple(
+        range(cube.ndim - len(cell_measure_cube.shape), cube.ndim)
+    )
+    cell_measure_data = cell_measure_cube.core_data()
+    if cell_measure_cube.has_lazy_data():
+        cube_chunks = tuple(cube.lazy_data().chunks[d] for d in coord_dims)
```

Comment:
we should really ask Iris folks to make the `chunks` object tuple available a lot easier than that :grin:

Reply:
I don't think it's that bad TBH, it's just one `lazy_data()` or `core_data()` more in the call. I am not sure how useful a `cube.chunks` would be, given that data can be non-lazy (chunks are not defined in this case) or that multiple arrays can be assigned with a cube (coordinates, cell measures, ancillary variables), each with their own chunking...

Reply:
it's be good to have it like `netCDF4.Dataset.chunks()` works - also a good comparison to that 0 the original HDF5 storage chunks vs iris rechunked data, but obv not in scope here. Thanks, Manu :beer: